src/scaladoc/scala/tools/nsc/doc/html/HtmlPage.scala

```diff
@@ -145,8 +145,12 @@ abstract class HtmlPage extends Page { thisPage =>
         <span class="extmbr" name={ mbr.qualifiedName }>{ inlineToHtml(text) }</span>
     case Tooltip(tooltip) =>
       <span class="extype" name={ tooltip }>{ inlineToHtml(text) }</span>
-    case LinkToExternal(name, url) =>
-      <a href={ url } class="extype" target="_top">{ inlineToHtml(text) }</a>
+    case LinkToExternalTpl(name, baseUrl, dtpl: TemplateEntity) =>
+      def url = Page.makeUrl(baseUrl, Page.templateToPath(dtpl)) + name
```

Comment:
Quoting too much won't hurt. <https://docs.oracle.com/javase/8/docs/api/java/net/URI.html#URI-java.lang.String-java.lang.String-java.lang.String-java.lang.String-java.lang.String-> knows how to do it correctly.